### ISSUE TITLE
Recall metric for classification

### ISSUE THREAD

**Issue by lars-reimann** [Safe-DS/Stdlib#187]
### Is your feature request related to a problem?

Computing the recall of a classifier for a specific class is a common task.

### Desired solution

* New method called `recall` in `Classifier`
* Required parameter `positive_class` to specify which value should be treated as positive (all other classes are treated as negative)
* Call [`sklearn.metrics.recall`](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.recall_score.html#sklearn.metrics.recall_score)
